VisualizationBase/src/overlays/MessageOverlay.cpp

```diff
@@ -32,11 +32,14 @@
 #include "../items/TextStyle.h"
 #include "../declarative/DeclarativeItem.hpp"
 
+#include "../VisualizationManager.h"
+
 namespace Visualization {
 
 DEFINE_ITEM_COMMON(MessageOverlay, "item")
 
-MessageOverlay::MessageOverlay(Item* associatedItem, const StyleType* style) : Super{{associatedItem}, style}
+MessageOverlay::MessageOverlay(Item* associatedItem, const StyleType* style, bool ignoresScaling)
```

Comment:
When I see an Item constructor that has an argument called `ignoresScaling`, I expect to find the `ItemsIgnoresTransformations` flag, but it's not there. How exactly does this overlay ignore scaling? Can't/shouldn't we make use of the flag?